src/schema/artist/__tests__/index.test.js

```diff
@@ -737,4 +737,45 @@ describe("Artist type", () => {
       })
     })
   })
+
+  describe("filtered artworks", () => {
+    it("returns filtered artworks", () => {
+      rootValue.filterArtworksLoader = sinon
```

Comment:
I've noticed from other PRs that due to Jest we've been moving away from sinon in favor of `jest.fn()`

Reply:
oh yea, whoops!